src/core/i18n/translation/locales/en.ts

```diff
@@ -22,12 +23,16 @@ export const translation: ITranslations = {
                 receive: 'Receive',
                 recover: 'Recover',
                 secure: 'Secure',
+                secureW: 'Secure wallet',
```

Comment:
we shouldn't add screen titles in labels. In labels we should add simple words that are reusable within the application

Reply:
also you could use `secureWallet` as key